docs/manual/webstreams/index.md

```diff
@@ -8,3 +8,5 @@ To add a web stream, click the **New** button on the right side of the Library p
 ![](static/Screenshot516-New_remote_webstream.png)
 
 The **Stream URL** setting must include the *port number* (such as 8000) and *mount point* (such as remote\_stream) of the remote stream, in addition to the streaming server name. A **Default Length** for the remote stream can also be set. If the stream is added at the end of a show which becomes overbooked as a result, it will be faded out when the show ends.
+
+Note: LibreTime checks the remote webstream's status upon editing stream settings, so an offline stream will result in an error. There are many tools such as [BUTT](https://danielnoethen.de/butt/) and [MIXXX](https://www.mixxx.org) that can be used to send a test stream to LibreTime can save it; read more [here](https://libretime.org/manual/live-shows-with-mixxx/).
```

Comment:
This link should be an relative reference, not to the libretime.org site

Reply:
Fixed.